### PR TITLE
Add Pekko Connectors Kafka download links

### PR DESCRIPTION
**Pull request by mdedetrich**

### CODE REVIEW

File: src/main/paradox/download.md [hunk 1]
```diff
@@ -21,6 +21,11 @@ These jars relate to the source downloads you find linked below.
...
 * [apache-pekko-http-1.0.0-incubating-src-20230726.zip](https://www.apache.org/dyn/closer.lua/incubator/pekko/HTTP-1.0.0/apache-pekko-http-1.0.0-incubating-src-20230726.zip) signature: ([asc](https://dlcdn.apache.org/incubator/pekko/HTTP-1.0.0/apache-pekko-http-1.0.0-incubating-src-20230726.zip.asc)) checksum: ([sha512](https://dlcdn.apache.org/incubator/pekko/HTTP-1.0.0/apache-pekko-http-1.0.0-incubating-src-20230726.zip.sha512))
 
+## Pekko Connectors Kafka
+
+* [apache-pekko-http-1.0.0-incubating-src-20230726.tgz](https://www.apache.org/dyn/closer.lua/incubator/pekko/CONNECTORS-KAFKA-1.0.0/apache-pekko-connectors-kafka-1.0.0-incubating-src-20230726.tgz) signature: ([asc](https://dlcdn.apache.org/incubator/pekko/CONNECTORS-KAFKA-1.0.0/apache-pekko-connectors-kafka-1.0.0-incubating-src-20230726.tgz.asc)) checksum: ([sha512](https://dlcdn.apache.org/incubator/pekko/CONNECTORS-KAFKA-1.0.0/apache-pekko-connectors-kafka-1.0.0-incubating-src-20230726.tgz.sha512))
```
Comment (pjfanning) [inline src/main/paradox/download.md:26]: the text in the name of name of the link is wrong - copy/pasted from the http links above

Comment (mdedetrich) [reply src/main/paradox/download.md]: Fixed and pushed.

File: content/download.html [hunk 2]
```diff
@@ -237,6 +240,11 @@ <h2><a href="#pekko-http" name="pekko-http" class="anchor"><span class="anchor-l
...
   <li><a href="https://www.apache.org/dyn/closer.lua/incubator/pekko/HTTP-1.0.0/apache-pekko-http-1.0.0-incubating-src-20230726.zip">apache-pekko-http-1.0.0-incubating-src-20230726.zip</a> signature: (<a href="https://dlcdn.apache.org/incubator/pekko/HTTP-1.0.0/apache-pekko-http-1.0.0-incubating-src-20230726.zip.asc">asc</a>) checksum: (<a href="https://dlcdn.apache.org/incubator/pekko/HTTP-1.0.0/apache-pekko-http-1.0.0-incubating-src-20230726.zip.sha512">sha512</a>)</li>
 </ul>
+<h2><a href="#pekko-connectors-kafka" name="pekko-connectors-kafka" class="anchor"><span class="anchor-link"></span></a>Pekko Connectors Kafka</h2>
+<ul>
+  <li><a href="https://www.apache.org/dyn/closer.lua/incubator/pekko/CONNECTORS-KAFKA-1.0.0/apache-pekko-connectors-kafka-1.0.0-incubating-src-20230726.tgz">apache-pekko-http-1.0.0-incubating-src-20230726.tgz</a> signature: (<a href="https://dlcdn.apache.org/incubator/pekko/CONNECTORS-KAFKA-1.0.0/apache-pekko-connectors-kafka-1.0.0-incubating-src-20230726.tgz.asc">asc</a>) checksum: (<a href="https://dlcdn.apache.org/incubator/pekko/CONNECTORS-KAFKA-1.0.0/apache-pekko-connectors-kafka-1.0.0-incubating-src-20230726.tgz.sha512">sha512</a>)</li>
```
Comment (pjfanning) [inline content/download.html:245]: * these links still have text saying `apache-pekko-http-1.0.0-incubating-src-20230726.`
* can you run `sbt paradox` again and copy the results into the `contents` dir?

Comment (mdedetrich) [reply content/download.html:245]: Woops I forgot, I will do this when I get back from lunch

Comment (mdedetrich) [reply content/download.html]: Fixed and pushed.

**Review by pjfanning — APPROVED** [review]
lgtm

**Pull request merged**